src/test/java/foodwhere/logic/parser/ParserUtilTest.java

```diff
@@ -37,7 +38,7 @@ public void parseIndex_invalidInput_throwsParseException() {
 
     @Test
     public void parseIndex_outOfRangeInput_throwsParseException() {
```

Comment:
I think this needs some test cases with whitespace to be trimmed, like `WHITESPACE + "1" + WHITESPACE`.